### PR TITLE
Improved Virtual Satellite Deployment

### LINKED ISSUE #125: Improve deployment and travis.yml
Currently there is no packed Virtual Satellite @ sourceforge. And travis.yml is groving too big.

- [x] Would be great if a downloadable packe exists as well.
- [x] Would be great to clean up and reduce travis.yml size.
- [x] Attach pdf manuals to github releases as well.

### PR DESCRIPTION
**Pull request by ghost**
- Now deploying executable binaries to sourceforge as well
- Added manuals to be deplyoed to GitHub Releases
- Cleaned up travis.yml and created additional bash scripts for build

Closes #125 .

### CODE REVIEW

File: bash/maven_build.sh [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	(grep -n "\[\(WARN\|ERROR\)\]" maven.log || exit 0  && exit 1;)
+	mvn install -P ${MAVEN_PROFILE},surefire,product -B -V | tee maven.log
+	echo "Check for Maven Problems on Product:"
+	(grep -n "\[\(WARN\|WARNING\|ERROR\)\]" maven.log || exit 0  && exit 1;)
+	echo "Check for failed test cases:"
+	(grep -n "<<< FAILURE!" maven.log || exit 0 && exit 1;)
+	echo "Ant jacoco Reports"
+	ant jacocoPrepareDependencies
+	ant jacocoReport 2>&1 | tee ant.log
+	(grep -n "Rule violated" ant.log || exit 0 && exit 1;)
```
Comment (franzTobiasDLR) [inline bash/maven_build.sh]: The build can also fail here if a e.g. a project does not have a source directory... Then surefire doesn't know how to handle the project. If that happens reports are not evaluated and the build succeeds without checking the coverage. "grep -n "Rule violated\|BUILD FAILED" ant.log" solves this. I have a fix for that on the requirements branch and can merge it later... Just to let you know ;)

**Review by franzTobiasDLR — DISMISSED** [review]
Nice!

File: bash/maven_build.sh (hunk 1 above)
Comment (PhilMFischer) [reply bash/maven_build.sh]: Good idea. I will include that suggestion.

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Comment by PhilMFischer** [comment]
@franzTobiasDLR Please check your dismissed review.

**Review by franzTobiasDLR — APPROVED** [review]

**Pull request merged**